### PR TITLE
Docs/Added Contribution Guidelines

### LINKED ISSUE #604: Add onboarding materials
We need to add tutorials / guides for the community to start contributing to the project

### PR DESCRIPTION
**Pull request by RakshitKhajuria**
# Description

**Added guides for the community to start contributing to the project**

-------------------------------------------------------------------------------------------------
➤ Closes #604

### CODE REVIEW

**Review by ArshaanNazir — APPROVED** [review]
LGTM. We can have a blog for explaining one end-to-end implementation and point to that as well ( through website as well as README)

**Comment by RakshitKhajuria** [comment]
> LGTM. We can have a blog for explaining one end-to-end implementation and point to that as well ( through website as well as README)

We have already provided an example in the website, it will be kind of similar thing for blog too

File: docs/pages/docs/contribute.md [hunk 1]
```diff
@@ -0,0 +1,92 @@
...
+            """
+            #Your code here
+            
+        def yourfunction(text):
+            """
+            Docstrings
+            """
+            .
+            .
+            .
```
Comment (RakshitKhajuria) [inline docs/pages/docs/contribute.md:42]: here

Comment (ArshaanNazir) [reply docs/pages/docs/contribute.md:42]: I mean one of the new robustness test can be explained. Not just sample code.

**Pull request merged**